app/sentry-example-page/page.tsx

```
@@ -0,0 +1,236 @@
+            fill="currentcolor"
+          />
+        </svg>
+        <h1>sentry-example-page</h1>
+
+        <p className="description">
+          Click the button below, and view the sample error on the Sentry{' '}
+          <a target="_blank" href={sentryIssuesUrl}>
+            Issues Page
+          </a>
+          . For more details about setting up Sentry,{' '}
+          <a target="_blank" href="https://docs.sentry.io/platforms/javascript/guides/nextjs/">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)